playbooks/roles/certs/defaults/main.yml

```diff
@@ -79,8 +80,10 @@ certs_git_identity: "{{ certs_app_dir }}/certs-git-identity"
 certs_requirements_file: "{{ certs_code_dir }}/requirements.txt"
 certs_version: 'rc'
 certs_templates_version : 'rc'
+certs_templates_repo: ''  # Set this to a private git repository of certificate templates
 certs_gpg_dir: "{{ certs_app_dir }}/gnupg"
 certs_template_dir: "{{ certs_app_dir }}/templates"
+certs_template_dir: "{{ certs_code_dir }}"
```

Comment:
We shouldn't be redefining this;
do we still need/want this change?

I suspect not...